test/dotnet-new.Tests/NewCommandTests.cs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using FluentAssertions;
+using Microsoft.DotNet.Tools.Test.Utilities;
+using System;
+using System.Linq;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.DotNet.New.Tests
+{
+    public class NewCommandTests
+    {
+		private readonly ITestOutputHelper output;
+
+		public NewCommandTests(ITestOutputHelper output)
+		{
+			this.output = output;
+		}
+
+        [Fact]
+        public void WhenSwitchIsSkippedThenItPrintsError()
+        {
+            var cmd = new DotnetCommand().Execute("new Web1.1");
```

Comment:
My concern is that it will just add more work for #5430 to get merged in.  Where if we picked a different command and switch, we wouldn't have to rewrite this test when #5430 is merged.